docs/quick_start.md

```diff
@@ -180,19 +158,19 @@ Note: *If any changes are made to the dependencies section of the `package.json`
     "test": "react-scripts test",
     "prebuild": "electron-rebuild",
     "build": "react-scripts build",
-    ...    
+    ...
 }
 ```
 
 The scripts section contains a number of commands you can run.
 
 They are also presented in key/value pair where the key is the command name and value is the command we want to run.
 
-*Checkout the [NPM Scripts](https://brown-ccv.github.io/honeycomb-docs/docs/npm_scripts) page for more information about the different scripts you can run.*
+_Checkout the [NPM Scripts](https://brown-ccv.github.io/honeycomb-docs/docs/npm_scripts) page for more information about the different scripts you can run._
 
 ### 4. Run the task in dev mode
 
-*Windows users must use a git bash terminal*
+_Windows users must use a git bash terminal_
```

Comment:
Maybe we can say for windows users we'd recommend using the git bash terminal. Do they actually have to use it?

Reply:
They have to use it, at least with the the scripts are set up right now. A bunch of the scripts use the `sleep` command which isn't available on Windows (command line or PowerShell) so the scripts will crash. Git bash is installed when you install git and uses all of the unix commands

Reply:
We could update the scripts in the honeycomb repo to work cross platform but I'm not really sure it'd be worth it? Would be a lot of testing and I'm not really sure what the end product is supposed to look like

Reply:
Ah I see okay no worries !